workflow_creation/utilities_plm_webclient.md

```diff
@@ -1,41 +1,228 @@
+
+## Why use PLM
+
+PLM provides, in a product-structure-based way, management of system, product and project data. Management of data means, among others, controlled sharing and integration of data as they emerge during a system/product/project life-cycle, and long-term archiving. Such data shall be exposed through a PLM Server to other users. Besides the data themselves, also meta data are made available that describe who created or modified the data, when, how and for what purpose. Also version and configuration data are such meta data.
+The remainder of this document refers to these three types of data, system, product and project data, as just “product data”.
+
+### Configuration control
+
+The PLM Server enables configuration control of the different versions of a product structure and of the data related to it. For this, the PLM Server provides, among others, the following functionality:
+- Create folder versions
+- Create baselines
+- Create folder versions of folders belonging to a baseline
+- Copy a folder version to another parent
+- Assign editors and readers to folder versions
+- Link versions of different folders to one another
+- Remove folder versions.
+
+### Persistence and traceability
+
+PLM data is always related to the product structure tree, and is presented in such a tree structure.
+
+<p align="center">
+  <img src="img_plm/product_structure_tree.png"
+   alt="Example of a product structure tree" />
+</p>
+
+The PLM Server allows you to store - in such tree - product data that are in parameters or that are in files, including documents, software, manuals, structured documents like STEP, etc.
+Versioning of folders in the product structure allows the history of the product data to be tracked throughout the development of the product.
+
+### Export and archival
+
+Data in the PLM Server will represent a rather comprehensive product description after having been collected over a period of time. For products with a long lifetime it is worthwhile to archive this data set. The PLM Server offers archiving capabilities in a standard representation (ISO 10303-209) for long term storage.
+
+## Roles of PLM in CAxMan workflows
+
+PLM functionality is available in CAxMan – and else – as web-services and through a web-client.
+The web-client user’s guide is in [User guide PLM client](./utilities_plm_webclient.md#user-guide-plm-client). The PLM web-client can act as a service within a CAxMan workflow to hand-over files to subsequent services. The invocation of the web-client as part of a CAxMan workflow is described in [this document](./utilities_plm_webclient.md#integrating-the-plm-web-client-in-a-workflow).
+
+Access to the PLM Server via web-services is twofold:
+1.	[Via GSS](../service_implementation/advanced_plm.md#gss).
+1.	[Via EDMopenSimDM native web-services](../service_implementation/advanced_plm.md#native-services--soap-).
+
+These two types of services give you access to, among others, the following functions:

```

Comment:
Here, also, I'd like to see "GSS" replaced by "PLM API" or similar. I understand that there is a difference between the native PLM API and the "PLM API for GSS", but I still think calling the latter part GSS only is wrong.

Reply:
I understand the point.
@JochenHaenisch , any idea on what should we use instead?

Reply:
What about "PLM access through GSS" as title instead of just "GSS"?

Reply:
I don't think that's correct either. The text behind the link does not explain anything about the usage of GSS, but the usage of the PLM API.

Reply:
I don't think that's necessary here. File access through GSS is documented, and from that perspective, there is no difference between PLM and any other storage provider.

Reply:
New proposal:
for the first title (GSS): PLM API through GSS
for the second title (native ...): PLM API through native services.

Reply:
Hmm, could you explain what you mean with "through GSS"?
If it means that obtaining the PLM endpoints is _through GSS_ and the rest is _not_ through GSS, I'm fine with it. ;-)

Reply:
OK - I have changed the titles and added some more introductory text to both sections. Alexey needs to commit this.
If you are happy with that, will you merge this into our training repository and inform people of the updates?
Thanks and Happy Easter, Jochen

Reply:
Sorry for my long silence, I hope you had a good Easter time.

Thanks, Jochen, for your contributions, I'm fine with the current status.